src/utils.rs

```diff
@@ -0,0 +1,7 @@
+use std::fmt::Display;
+
+use napi::{Error, Status};
+
+pub fn js_generic_err(msg: impl Display) -> Error<Status> {
+  Error::new(Status::GenericFailure, msg)
+}
```

Comment:
I'm happy to see an instant use of new knowledge from today's lab - `impl Display` :wink: